projects/ngx-scrollbar/src/lib/utils/resize-sensor.directive.ts

```diff
@@ -12,7 +12,7 @@ import { NgScrollbar } from '../ng-scrollbar';
  */
 @Injectable({ providedIn: 'root' })
 export class ResizeObserverFactory {
-  private readonly resizeObserverSource = new BehaviorSubject(null);
+  private readonly resizeObserverSource = new BehaviorSubject<any>(null); // TODO define correct type here
```

Comment:
`any` should be used because the original interface is not available in SSR

Reply:
Ok